banking/snapshot_test.go

```diff
@@ -112,7 +113,7 @@ func TestSnapshotRoundtripViaEngine(t *testing.T) {
 
 	now = time.Now()
 	deliver := now.Add(time.Hour)
-	eng.OnTick(ctx, now)
+	eng.OnTick(ctx, time.Time{})
```

Comment:
probably doesn't matter but why are you not passing `now`?

Reply:
Because it is not used anymore inside `OnTick`

Reply:
No, in the normal use we do not use the time in that function in most of the places already.
It is in the list of parameters because this signature is expected as a callback, and not all engines have the time value removed yet (and the interfaces are not reworked on that level with this PR).
I removed it everywhere because in the realtime version of these places, time paramer is `_`, so it does not make sense according to me to send it anyway.

Reply:
I don't think it makes sense. Either the contract is that we pass the current time to the callback, or we drop the time from the callback.

Reply:
what the services do with the time is not the point. My point is that if the contract of the callback is that it is passed the current time, then I don't see why we'd do something else in the test. The fact that the actual implementation of the callback doesn't care about the parameter being passed is not relevant.

Reply:
Ok

Reply:
This has been fixed on all places https://github.com/vegaprotocol/vega/pull/5475/commits/8e6d1e25f8e5bd6a121808ea8d58fb135f37f449
Thanks!